podcasts/WhatsNewPageView.swift

```diff
@@ -91,11 +91,11 @@ class WhatsNewPageView: ThemeableView {
                     let button = WhatsNewLinkButton(url: url)
                     button.setTitle(Bundle.main.localizedString(forKey: text, value: text, table: nil), for: .normal)
                     button.titleLabel?.font = UIFont(name: "SFProText-Medium", size: 18)
-                    button.titleLabel?.textAlignment = .left
+                    button.titleLabel?.textAlignment = .natural
```

Comment:
This is not used anymore so these changes are ok. I wrote it down so we can remove this in the future.